src/services/data/documents.ts

```diff
@@ -10,14 +10,16 @@ export default (
   httpAuth: HttpInstance
```

Comment:
Since it is a node-service that only has the response conversion, we don't need to adapt end points that only have custom request data

Reply:
Or do we want to set them anyway for if we move in the future?

Reply:
👌 If we don't do this now I think a note in the ticket to move to the requestInterceptor would be helpful though

Reply:
https://extrahorizon.atlassian.net/browse/SVCS-2583